### PR TITLE
perf(ivy): introduce new benchmark for view create, destroy, traverse

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**

### CODE REVIEW

**Review by kara — APPROVED** [review]
LGTM, one nit

**Comment by kara** [comment]
[presubmit](http://test/OCL:259754193:BASE:259754225:1563985149698:b62b2c39)

File: modules/benchmarks/src/views/index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+<body>
+
+  <h1>Views traversal benchmark</h1>
+
+  <benchmark-root>loading...</benchmark-root>
+
+  <script>
+    addEventListener('DOMContentLoaded', () => {
+      // DevServer has automatic bootstrap code, so if we already have <scripts> than we don't need to bootstrap
+      var alreadyBootstraped = document.querySelectorAll('script').length > 1; // 1 for ourselves
```
[the remark this answers is not in the document]
Comment (pkozlowski-opensource) [reply modules/benchmarks/src/views/index.html]: Done

**Review by kara — APPROVED** [review]
LGTM

**Comment by kara** [comment]
merge-assistance: global approval

**Pull request closed without merging**